ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -189,6 +189,22 @@ objects:
         done
         # oh god the blood
         sleep 180
+        # temporary - remove this check once a proper test is added to origin e2e
+        password=$(cat /tmp/artifacts/installer/auth/kubeadmin-password)
+        TARGET="$(date -d '5 minutes' +%s)"
+        NOW="$(date +%s)"
+        while [[ "${NOW}" -lt "${TARGET}" ]]; do
+          # wait until oc login -n kubeadmin succeeds
+          if oc login -u kubeadmin -p "${password}"; then
+            if ! oc login -u system:admin; then
```

Comment:
Is this a check to see if the `kubeadmin` login granted you sufficient authorization?

Reply:
This is to set user back to system:admin for the test suite, to keep it as it was before this check.